lib/health_cards/verifiable_credential.rb

```diff
@@ -44,6 +47,14 @@ def credential
 
     def jwt; end
 
+    def save_as_qrcode
```

Comment:
Can we add a filepath argument here and then have `QR_CODE_PATH` be the default?